client/src/components/FooterButton.js

```diff
@@ -17,7 +17,7 @@ export const FooterButton = styled.button`
   cursor: pointer;
   &:hover {
     color: #ffffff;
```

Comment:
Just a minor issue. In some components I can see that you have a mix of colors coming from your `theme.js` and some values are defined static in the component file (example `#ffffff`). If every color comes from the theme file, you make later changes in the colors of your app easier.